datagrepper.spec

```diff
@@ -0,0 +1,278 @@
+# https://bugzilla.redhat.com/show_bug.cgi?id=955781
+%global pypi_name datagrepper
+%global pypi_version 1.0.0
+
+Name:           %{pypi_name}
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        A webapp to query fedmsg history
+
+License:        GPLv2+
+URL:            https://github.com/fedora-infra/datagrepper
+Source0:        %{pypi_source %{pypi_name}}
+
+BuildArch:      noarch
+
+BuildRequires:  make
+BuildRequires:  python3-arrow
+BuildRequires:  python3-datanommer-models
```

Comment:
Do you actually need to specify these buildrequires since you use `%generate_buildrequires`?

Reply:
Good point, I don't think so. Everything that is in pyproject.toml as a dependency should not need to be here, right?

Reply:
Yes, you only need to keep the development dependencies for the unit tests to pass. You can check the output of rpmbuild and the resulting SRPM to see if the deps are there.